mono/metadata/sgen-mono.c

```diff
@@ -2799,18 +2800,19 @@ mono_gc_add_memory_pressure (gint64 value)
  */
 
 void
-sgen_client_degraded_allocation (size_t size)
+sgen_client_degraded_allocation (void)
```

Comment:
Anything starting with `sgen_client_` is part of the SGen API, so this is a breaking change.